code/modules/antagonists/traitor/components/demoraliser.dm

```diff
@@ -44,8 +44,13 @@
 	// If you're not conscious you're too busy or dead to look at propaganda
 	if (viewer.stat != CONSCIOUS)
 		return
+	if(viewer.is_blind())
+		return
 	if (!should_demoralise(viewer))
 		return
+	if(!viewer.can_read(host, (moods.text_based ? (READING_CHECK_LITERACY|READING_CHECK_LIGHT) : (READING_CHECK_LIGHT)), TRUE)) //if it's a text based demoralization datum, make sure the mob has the capability to read. if it's only an image, make sure it's just bright enough for them to see it.
```

Comment:
I'm not a fan of this, would it be better if ``text_based`` var held the flags instead? so it would be something like

``var/reading_requirements = READING_CHECK_LITERACY | READING_CHECK_LIGHT``

Reply:
I dig it.